llvm/test/tools/yaml2obj/GOFF/GOFF-no-header.yaml

```diff
@@ -0,0 +1,6 @@
+# RUN: not yaml2obj %s FileCheck --ignore-case %s --check-prefix=ERR
+
+## X: [] is an extra field is to workaround document of unknown type error
+# ERR: yaml2obj: error: missing required key 'FileHeader'
+--- !GOFF
+X: []
```

Comment:
There's no need to use `--check-prefix` as there is only one prefix in this file. Just use `CHECK`.

Also, I'd reorder the test a little as follows, as I think this aids readability:
```
# RUN: ...
# CHECK: ...

--- !GOFF
## X: [] is an extra field ...
X: []
```

Reply:
Done